src/PowerShellEditorServices/Session/PowerShellContext.cs

```diff
@@ -796,7 +797,7 @@ public async Task ExecuteScriptWithArgs(string script, string arguments = null,
                 if (File.Exists(script) || File.Exists(scriptAbsPath))
                 {
                     // Dot-source the launched script path
-                    script = ". " + EscapePath(script, escapeSpaces: true);
+                    script = ". " + QuoteEscapeString(script);
```

Comment:
Will this continue to work if someone has a debug launch config like this:
```
"script": "${workspaceFolder}/foo`[bar`].ps1"
```
I "think" if we single quote a path, we have to evaluate backticks as well and perhaps strip some out.  For instance, with the above path the user tried to be helpful and escape the wildcard chars for us (even though unnecessary, this works today).  If the backticks are left in, the path will fail with this change - I think.  However take a file named `foo`bar.ps1` and it requires the backtick as a literal char.

Reply:
I'll give it a test. I don't think we can do both though. Even if it means a breaking change, personally my preference is to enable all paths to work rather than having some paths work two ways and others not work at all.

It feels like we should treat the config setting as a `LiteralPath` -- it's possible to call a file ``foo`[bar`].ps1`` on the filesystem (I experimented with that while making the fix). In fact on Windows it's not possible to make `*.ps1` or `?.ps1` but on *nix it is, and have `` `?.ps1`` right next to them in the directory.

Reply:
Worth noting here too that `Set-PSBreakpoint` does not have a `-LiteralPath` parameter -- something which there is a desire to fix I think.

Reply:
@rkeithhill I just tested with two files in the same directory (`[script].ps1` and `` `[script`].ps1``) and in 1.9.0, backticks or not both hit the first file.

Unfortunately the same happens in 1.9.1, so looks like I still need to do some work in this PR.

Reply:
So I think I'm hitting a powershell issue here. I'll look into a workaround and open an issue there

Reply:
I've found something interesting. PowerShell behaves strangely with backtick escaping: https://github.com/PowerShell/PowerShell/issues/7999.

It looks like we've always been receiving the message in the first place with the backtick escaped but not other places:
```
2018-10-11 11:31:44.987 [DIAGNOSTIC] C:\Users\roholt\Documents\Dev\PowerShellEditorServices\src\PowerShellEditorServices.Protocol\MessageProtocol\MessageReader.cs: In method 'ReadMessage', line 114:
    READ MESSAGE:
    
    {
      "command": "launch",
      "arguments": {
        "type": "PowerShell",
        "request": "launch",
        "name": "Launch Script",
        "script": "C:\\Users\\roholt\\Documents\\Dev\\sandbox/``[script``].ps1",
        "args": [],
        "cwd": "C:\\Users\\roholt\\Documents\\Dev\\sandbox",
        "createTemporaryIntegratedConsole": false,
        "internalConsoleOptions": "neverOpen",
        "__sessionId": "ecbc3316-e3f6-48ea-a3cf-dc6730758edc"
      },
      "type": "request",
      "seq": 2
    }
```

This miraculously worked because PowerShell escapes `` ````[ `` the same as it escapes `` `[ ``, which I think is a bug.

So I'm currently trying to track down why the message we get doubles the backticks.